src/Features/Core/Diagnostics/DiagnosticData.cs

```diff
@@ -80,7 +81,7 @@ internal sealed class DiagnosticData
                     severity, severity, isEnabledByDefault, warningLevel,
                     ImmutableArray<string>.Empty, ImmutableDictionary<string, string>.Empty,
                     workspace, projectId, documentId, span,
-                    originalFilePath, originalStartLine, originalStartColumn, originalEndLine, originalEndColumn,
+                    null, originalStartLine, originalStartColumn, originalEndLine, originalEndColumn,
```

Comment:
I don't understand this - why are we passing null here and not originalFilePath?